charts/tileserver-api/templates/deployment.yaml

```
@@ -0,0 +1,85 @@
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "tileserver.name" . }}
+  labels:
+    {{- include "api.labels" . | nindent 4 }}
+
+spec:
+  replicas: {{ .Values.tileserver.api.replicaCount }}
+  revisionHistoryLimit: {{ .Values.tileserver.api.revisionHistory }}
+
+  selector:
+    matchLabels:
+      {{- include "api.selectorLabels" . | nindent 6 }}
+
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 1
+
+  template:
+    metadata:
+      labels:
+        {{- include "api.selectorLabels" . | nindent 8 }}
```

Тут стоит дать возможность пользователю добавлять свои label'ы и аннотации.

```suggestion
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}
      labels:
        {{- include "api.labels" . | nindent 8 }}
      {{- with .Values.podLabels }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
```